spec/spec_helper.rb

```diff
@@ -10,6 +10,26 @@
 require 'bundler'
 Bundler.require :default, :test
 
+# Grape uses autoload https://api.rubyonrails.org/classes/ActiveSupport/Autoload.html.
```

Comment:
Let's put this in https://github.com/ruby-grape/grape/tree/master/spec/support, call it `eager_load.rb` and call this function `eager_load!` to follow the Ruby pattern.